### DISCUSSION TITLE
Quasar's auto scroll creator (don't know how to call it) is duplicating my page for some reason?

### DISCUSSION THREAD

**Discussion by linomass36** [quasarframework/quasar#12829] [General - Components / Directives / etc]
What's in the title, my Layout code has:
```
<q-page-container>
 <router -view />
</q-page-container>
```

so I don't know what might cause this:

https://user-images.githubusercontent.com/87259635/158758278-41bf6b16-b297-434b-adec-1891596e1b73.MOV

**Comment by MilosPaunovic** [comment]
This seems like an issue with your code. Plese provide a reproduction of some sort for someone to be able to take a closer look.

**Comment by linomass36** [comment]
The page's code:

[https://codepen.io/linomass36/pen/xxpZKWd](url)

**Reply by MilosPaunovic** [reply]
The link above is not a valid reproduction. Please fix it, or create a minimal reproduction repo.

**Reply by linomass36** [reply]
done, i think?

**Reply by MilosPaunovic** [reply]
Nope, try to open it in new tab. It shows blank screen. By reproduction I don't mean only to paste code, it has to work (to the point of your issue) so someone could easily check it out and guide you to best solution.
